airflow/providers/databricks/operators/databricks.py

```diff
@@ -1066,3 +1083,6 @@ def execute(self, context: Context) -> None:
         self.launch_notebook_job()
         if self.wait_for_termination:
             self.monitor_databricks_job()
+
+    def execute_complete(self, context: dict | None, event: dict):
+        _handle_deferrable_databricks_operator_completion(event, self.log)
```

Comment:
It seems this function is not doing the same thing as its sync version 🤔

Reply:
Both do print similar logs. But, I do see there's a difference. Does it need to be exactly the same implementation? If so, I can update it.

Although, it will raise error because of `event["repair_run"]` and I will push a fix to use get so that it won't fail

Reply:
> Both do print similar logs. But, I do see there's a difference. Does it need to be exactly the same implementation? If so, I can update it.

Ideally, yes. If it does not make the code super complicated.

> Although, it will raise error because of event["repair_run"] and I will push a fix to use get so that it won't fail

Yep, `repair_run` is something I worried about.

Reply:
@Lee-W , I updated execute_complete to be the same as the sync version.